lib/seafoam/graal/source.rb

```diff
@@ -0,0 +1,23 @@
+module Seafoam
+  module Graal
+    # Routines for understanding source positions in Graal.
+    module Source
+      def self.render(source_position)
+        lines = []
+        caller = source_position
+        while caller
+          method = caller[:method]
+          lines.push '  ' + render_method(method)
```

Comment:
Is the indent important? It makes `#source` a little harder to test (the `gsub(/^\s*/, '')` business) and could be mildly inconvenient if you wanted to pipe the output of `seafoam source` elsewhere.

Reply:
Fixed.